projects/netgrif-components-core/src/lib/tabs/classes/opened-tab.ts

```diff
@@ -57,6 +57,11 @@ export class OpenedTab implements TabContent {
      */
     public tabClosed$: Subject<void>;
 
+    private _labelIcon$: BehaviorSubject<string | undefined>;
```

Comment:
You should create new class for tab label with icon and text as properties

Reply:
Done